packages/block-editor/src/components/block-styles/index.js

```diff
@@ -125,12 +123,15 @@ function BlockStyles( {
 					>
 						<div className="editor-block-styles__item-preview block-editor-block-styles__item-preview">
 							<BlockPreviewContent
-								name={ name }
-								attributes={ {
-									...attributes,
-									className: styleClassName,
-								} }
-								innerBlocks={ block.innerBlocks }
+								blocks={
+									{
```

Comment:
The "cloneBlock" already takes care of merging attributes and also safely clones innerBlocks. It feels like it's well-suited for this usage here. (Same for the switcher and inserter menu)

Reply:
Thanks. That's much cleaner.